lib/osf-components/addon/components/contributors/card/editable/template.hbs

```diff
@@ -51,7 +51,22 @@
                 {{on 'click' (fn @manager.toggleContributorIsBibliographic @contributor)}}
             />
         </span>
-        <span local-class='CardSectionSmall'>
+        <span
```

Comment:
Was this only on the read-only card before, not the editable card? If so, this seems like a good thing to add to the "changes" section of the PR description. If not, I'm very confused why it's here.

Reply:
Correct. This was not in the editable cards. I've added to the changes in the PR description.